packages/cos-setup/definition.yaml

```
@@ -1,7 +1,7 @@
 name: cos-setup
 category: system
-version: 0.4-12
+version: 0.4-13
```

Let's leave revbumps for the bot unless strictly necessary

```suggestion
version: "0.5"
```